### PR TITLE
Avoid missing constants by using autoload

### PR DESCRIPTION
**Pull request by kke**
Needed to make current versions of plugins run with 1.3.0

Before:
```
$ bin/kontena master ssh
 [error] NameError : uninitialized constant Kontena::Cli::ShellSpinner
```

After:
```
$ bin/kontena master ssh
Works
```

### CODE REVIEW

File: cli/examples/kontena-plugin-hello/kontena-plugin-hello.gemspec [hunk 1]
```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |spec|
   spec.description   = "This plugin is just an example"
   spec.homepage      = "https://github.com/kontena/kontena/cli/examples/kontena-plugin-hello"
 
-  spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+  spec.files         = Dir['**/*']
```
Comment (jakolehm) [inline cli/examples/kontena-plugin-hello/kontena-plugin-hello.gemspec]: Why?

Comment (kke) [inline cli/examples/kontena-plugin-hello/kontena-plugin-hello.gemspec]:
```
$ gem build kontena-plugin-hello.gemspec
fatal: Not a git repository (or any of the parent directories): .git

WARNING:  licenses is empty, but is recommended.  Use a license identifier from
http://spdx.org/licenses or 'Nonstandard' for a nonstandard license.
WARNING:  prerelease dependency on kontena-cli (>= 1.0.0.pre2) is not recommended
WARNING:  See http://guides.rubygems.org/specification-reference/ for help
  Successfully built RubyGem
  Name: kontena-plugin-hello
  Version: 0.1.0
  File: kontena-plugin-hello-0.1.0.gem
$ gem unpack kontena-plugin-hello-0.1.0.gem
Unpacked gem: '/private/tmp/hello/kontena-plugin-hello-0.1.0'
$ ls -al /private/tmp/hello/kontena-plugin-hello-0.1.0
total 0
drwxr-xr-x   2 user  wheel   68 Jun  7 16:13 .
drwxr-xr-x  10 user  wheel  340 Jun  7 16:16 ..
```

**Comment by jnummelin** [comment]
@kke Could you do a rebase/merge with master. #2405 was merged and makes testing this PR much easier. :)

**Comment by kke** [comment]
Rebased

**Comment by jnummelin** [comment]
Hmm, I don't see #2405 stuff in. :/ Did you push after rebase?

**Comment by kke** [comment]
Now.

**Comment by jnummelin** [comment]
Seems to fix the issue, at least DO plugin now is able to create both master and node.

**Comment by kke** [comment]
master ssh to vagrant was broken for me without this.

**Comment by jnummelin** [comment]
I think we need to try out every single plugin to be sure we donät have any more nasty surprises

**Comment by jnummelin** [comment]
@jakolehm Care to have another look into this?

**Pull request merged**